### PR TITLE
fixes #138

### LINKED ISSUE #138: STOW-RS triplet overwrite capability
Currently, when a C-STORE (DIMSE) request is converted to a STOW-RS (HTTP) request, it will fail with an [HTTP 409](https://cloud.google.com/apis/design/errors#handling_errors) error code if the UID triplet (studyUID/seriesUID/instanceUID) already exits in the DICOM store.

This deviates from the expectation for several PACS systems, which would expect the destination system to accept incoming instances and replace those stored.

By adding this configuration option, the adapter can be configured to internally perform a HTTP POST -> (HTTP 409) -> HTTP DELETE -> HTTP POST, which will effectively delete and replace the instance with the new instance that is being ingested.

### PR DESCRIPTION
**Pull request by jasonklotzer**

### CODE REVIEW

**Review by danielbeaudreau — CHANGES_REQUESTED** [review]
Thanks for the contribution.

File: import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java [hunk 1]
```diff
@@ -163,13 +158,17 @@ private static IDicomWebClient configureDefaultDicomWebClient(
       Flags flags) {
     IDicomWebClient defaultCstoreDicomWebClient;
     if (flags.useHttp2ForStow) {
```
[the remark this answers is not in the document]
Comment (jasonklotzer) [reply import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/ImportAdapter.java:160]: Yes, no HTTP2 being used by this user.

**Comment by jasonklotzer** [comment]
Closing and moving this PR to a native branch with updates.

**Pull request closed without merging**